foundation.dd

```diff
@@ -63,7 +63,9 @@ $(H3 Official contacts)
 $(P The D Language Foundation has been incorporated with the state of Washington, USA.
     The employer ID is 47-5352856.)
 
-$(P If you want to get in touch with the D Language Foundation, please feel free to write to $(LINK2 mailto:foundation@dlang.org, foundation@dlang.org).)
+$(P If you want to get in touch with the D Language Foundation, please feel free to write email at 
```

Comment:
nitpick: instead of "please feel free to write email" it's more correct to say "please feel free to write *an* email"